tests/test_estimators/test_quantile_regression.py

```diff
@@ -17,8 +18,9 @@
 
 def _create_dataset(coefs, intercept, noise=0.0):
     np.random.seed(0)
-    X = np.random.randn(10000, coefs.shape[0])
```

Comment:
going from `10_000` to `1000` was a big achievement to reduce runtime of the solvers